Rewrite basic-use.rst:

Mrt134 opened the pull request:
Complete rewrite of the file, including:
- Applied PEP8 conventions.
- Updated obsolete information and examples.
- Add index entries and internal reference points.
- Update section titles and levels.
- Rewrite sentences to fix grammar, improve clarity, spelling, etc.
- Add in Sphinx/.rst directives/syntax for examples, warning boxes, code-blocks, internal references, commands, and samples.
- Build tested against current iocage docs: no Sphinx errors, no build errors. The updated file should be able to drop right into the current doc structure.

Make sure to follow and check these boxes before submitting a PR! Thank you.

- [x] Explain the feature
- [x] Read [CONTRIBUTING.md](https://github.com/iocage/iocage/blob/master/CONTRIBUTING.md)

Mrt134 committed:
Rewrite basic-use.rst:

Complete rewrite of the file, including:
- Applied PEP8 conventions.
- Updated obsolete information and examples.
- Add index entries and internal reference points.
- Update section titles and levels.
- Rewrite sentences to fix grammar, improve clarity, spelling, etc.
- Add in Sphinx/.rst directives/syntax for examples, warning boxes, code-blocks, internal references, commands, and samples.
- Build tested against current iocage docs: no Sphinx errors, no build errors. The updated file should be able to drop right into the current doc structure.

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
 
-``iocage fetch release=9.3-RELEASE``
+ Type the number of the desired RELEASE
+ Press [Enter] to fetch the default selection: (default)
```
We should still show the default as `11.0-RELEASE`, so the user knows it is capable of autofilling with useful information and not just `default`.

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
+If a specific RELEASE is required, type:
 
-``iocage fetch ftpdir=/dir/``
+:samp:`# iocage fetch release=10.3-RELEASE`
```
this should be `iocage fetch -r 11.0-RELEASE`

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
-Create a jail
+If a specific download mirror is required, type:
+
+:samp:`# iocage fetch ftphost=ftp.hostname.org`
```
This is now `iocage fetch -s ftp.hostname.org`

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
+
+:command:`fetch` can also pull from a specific ftp directory:
+
+:samp:`# iocage fetch ftpdir=/dir/`
```
This is now `iocage fetch -d dir/`

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
-By default iocage will create a fully independent jail of the current host's RELEASE and set the TAG property to todays date.
+With a release downloaded, iocage is now able to create jails. There are
+five types of jails, three basic and two advanced. Basic jail types are
+**full**, **clone**, and **base** jails. Advanced jails types are
```
There are just normal ("full") and base now. I say "full" because it's not like the previous implementations full which is a fully standalone zfs dataset that is sent and received, it's instead a clone, but for the user, that implementation detail shouldn't matter.

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
+With a release downloaded, iocage is now able to create jails. There are
+five types of jails, three basic and two advanced. Basic jail types are
+**full**, **clone**, and **base** jails. Advanced jails types are
+**empty** and **templates**, but these are discussed in the
```
Template shouldn't be plural here.

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
+Depending on the user's requirements, the :command:`create` subcommand
+can be tweaked to create any of the three basic types. By default,
+iocage creates a fully independent jail of the current host's RELEASE,
+setting the TAG property to the current date.
```
Can only create two basic types, and by default creates a clone with the user specified RELEASE, not the hosts. The TAG property is being removed as we speak, so that part can be dropped.

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
+:samp:`# iocage create -r 11.0-RELEASE`
 
-Creating a jail is real simple, just run:
+This basic jail is fully independent.
```
Like I said up above, not independent, but I'd just drop the sentence. Not terribly useful anyways to the average user.

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
+This basic jail is fully independent.
 
-``iocage create``
+To create a full jails with multiple clones, type:
```
Should be stated as to create multiple jails. They're all clones, but using `--count` does not cause a relationship link between them. They are all individual clones from the RELEASE.

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
 
-To create a lightweight jail (clone) run:
+The numeric value after the **-c** flag is used to designate the number
+of clone jails to create. In the above example, *two* jails are created,
```
Just drop "clone"

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
-To create a lightweight jail (clone) run:
+The numeric value after the **-c** flag is used to designate the number
+of clone jails to create. In the above example, *two* jails are created,
+**one** *full* jail and **one** clone jail.
```
Drop this line, only two jails are created.

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
+specific jail **properties** can also be set. For example:
 
-To create a jail and set its IP address and tag property run:
+:samp:`# iocage create -r 11.0-RELEASE tag=myjail boot=on`
```
`TAG's` are being dropped, so for this example use `--name` instead.

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
+:samp:`# iocage create -r 11.0-RELEASE tag=myjail boot=on`
 
-``iocage create -c tag=myjail ip4_addr="em0|10.1.1.10/24"``
+Creates a FreeBSD 11.0 jail with the custom tag *myjail* and sets the
```
`11.0-RELEASE` we have to be consistent with nomenclature. s/tag/name/

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
+all jails, use :command:`iocage list` and read the **STATE** column.
 
-``iocage list -t``
+Use each jail's UUID or custom TAG to start, stop, or restart it. When
```
s/TAG/name/ The partial stuff should now apply to both name and UUID, so it should be reworded as Partial entries are acceptable.

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
 
-Start, stop or restart a jail
------------------------------
+.. tip:: Partial UUIDs can also be supplied for any other iocage
```
Partial entries instead as stated above.

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
 +++++
 
-To start a jail tagged www01 simply run:
+To start a jail with the custom tag **www01**, type:
```
With the name, no more TAGs :)

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
+:samp:`# iocage start www01`
 
-To start a jail with a full UUID run:
+A jail can also be started with a full UUID:
```
This can be elaborated with (default naming scheme) or something similar. As that will remain our default, unless the user specifies `--name`

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
+:samp:`# iocage start 26e8e027-f00c-11e4-8f7f-3c970e80eb61`
 
-Or to start the jail only with a partial UUID enter the first few characters only:
+A partial UUID is also acceptable:
```
entry

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
 
-Get property
-++++++++++++
+To set the TAG property for a jail (after creation):
```
Replace TAG with a different property, perhaps `notes` or `ip4_addr` or something.

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
+To set the TAG property for a jail (after creation):
 
-To verify any property simply run the ``get`` subcommand:
+:samp:`# iocage set tag=www02 26e8e027`
```
Same as above :)

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -1,153 +1,259 @@
 
-System wide defaults
---------------------
+:samp:`# iocage get tag 26e8e027`
```
s/tag/SOMETIHNGELSE/

skarekrow reviewed (changes requested):
Thanks!

Mrt134 committed:
Update basic-use.rst

- Address all review comments.
- Small adjustments to examples vs. commands
- Review and remove/replace all instances of "tag"

skarekrow commented on doc/source/basic-use.rst:
```diff
@@ -77,46 +78,43 @@ Create a Jail
+This normal jail is a clone of the specified RELEASE.
 
-To create a full jails with multiple clones, type:
+To create a multiple jails, use the **-c** option:
```
We can drop the `a`

Mrt134 committed:
Remove the "a" that got away... ;)
